src/ui/dialog-vkbasalt.ui

```diff
@@ -0,0 +1,819 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Created with Cambalache 0.10.2 -->
+<interface>
+  <!-- interface-name dialog-vkbasalt.ui -->
+  <requires lib="gtk" version="4.0"/>
+  <requires lib="libadwaita" version="1.0"/>
+  <template class="VkBasaltDialog" parent="AdwWindow">
+    <property name="default-width">550</property>
```

Comment:
It might be a good idea to add a default-height.

Reply:
What do you suggest?